frontend/src/component/NavBar.jsx

```
@@ -0,0 +1,63 @@
+      </div>
+
+      {/* Mobile Menu Overlay */}
+      {isOpen && (
+        <motion.div
+          initial={{ opacity: 0, x: "100%" }}
+          animate={{ opacity: 1, x: "0%" }}
+          exit={{ opacity: 0, x: "100%" }}
+          className="fixed inset-0 bg-black/90 text-white z-50 flex flex-col items-center py-12"
+        >
+          {/* Close Button */}
+          <button onClick={() => setIsOpen(false)} className="absolute top-6 right-6 text-gray-400 hover:text-white">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)